integrations/sonarqube/client.py

```diff
@@ -206,7 +217,7 @@ async def get_components(
 
         try:
             response = await self.send_paginated_api_request(
-                endpoint=Endpoints.PROJECTS,
+                endpoint=Endpoints.PROJECTS_INTERNAL,
```

Comment:
let's use the GA project API here for the get_components() and that's what will be used by new customer who install the integration

Reply:
pass endpoint as a parameter to `get_component` function, default to `PROJECTS`